examples/generate-files.sh

```
@@ -0,0 +1,478 @@
+#if [[ "$ACCESS" == "rox" ]]; then
+#    echo "${ACCESS_ROX_ACCESS_MODE}" >> kustomization.yaml
+#fi
+popd > /dev/null
+
+
+#
+# Generate the Yaml with all the objects
+#
+OUTPUT_FILENAME=${OUTPUT_DIR}/${NAME_SUFFIX}${ENV_FILENAME_SUFFIX}.yaml
+
+${KUSTOMIZE} build overlays/scope > ${OUTPUT_FILENAME} || exit 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)